articles/anomaly-detection/index.md

```diff
@@ -24,21 +24,21 @@ A **trigger** is a suspicious event that is detected when someone is trying to l
 
 If this block is triggered, it can be cleared the following ways:
 
-* An administrator removes the block via the Management Dashboard or API;
+* An administrator removes the block via the Dashboard (by clicking `unblock for all IPs` under the `ACTIONS` button when viewing the user's details) or by using the [Management API](https://auth0.com/docs/api/management/v2#!/User_Blocks/delete_user_blocks) ;
```

Comment:
Hi @Annyv2 ! Please make the following changes:

- Use relative links:
`/api/management/v2#!/User_Blocks/delete_user_blocks`
instead of `https://auth0.com/docs/api/management/v2#!/User_Blocks/delete_user_blocks`

- Add a link at `Dashboard` using the variable `${uiURL}`

- Replace the `unblock for all IPs` and `ACTIONS` with bold: __unblock for all IPs__ under the __ACTIONS__